src/javascript/model/imageData.js

```diff
@@ -0,0 +1,21 @@
+import { createApi } from 'unsplash-js';
+
+const unsplash = createApi({
+  accessKey: 'RmRQetkGdLG3o6Dv0LTcQ9-7Fvi6X1FoTF6Ax_pDsV4',
+  fetch: global.fetch,
+});
+
+const fetchImageData = async (queryText) => {
+  const result = await unsplash.search.getPhotos({
+    query: queryText,
+    page: 1,
+    perPage: 30,
+  });
+  const index = Math.floor(Math.random() * 30);
+  const img = result.response.results[index].links.download;
+  const avarageColorCode = result.response.results[index].color;
+  // console.log(">2> secondly run this");
```

Comment:
### Remove commented out code
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Done.